hilgendorf_moveit_config/config/joint_limits.yaml

```diff
@@ -0,0 +1,61 @@
+joint_limits:
+  right_wrist_2_joint:
+    has_velocity_limits: true
+    max_velocity: 3.2
+    has_acceleration_limits: false
+    max_acceleration: 0
```

Comment:
The moveit_config package here is posted, unmodified after running the setup assistant. If these are a problem, what is a reasonable value for max_acceleration? "3.5" as in the UR5_moveit_config package?